action-server/actions/__main__.py

```diff
@@ -0,0 +1,18 @@
+from actions.answers import QaResponse, QaProtocol
+import os, sys
+
+
+FAQ_URL_ENV_KEY = "faq_url"
```

Comment:
The environment variable name should be in uppercase and maybe a bit more explicit IMO, something like `COVID_FAQ_SERVICE_URL`